model.qmd

```diff
@@ -0,0 +1,114 @@
+
+Using Bayes’ theorem, Bayesian inference combines:
+
+  - **The likelihood**: the probability of observing the data given specific parameter values, and
+  - **The prior distribution**: our initial assumptions about the parameter values before observing the data.
+  
+The result is the **posterior distribution** of a parameter $\theta$ given data $y$:
+
+$$
+p(\theta | y) = \frac{p(y|\theta) p(\theta)}{p(y)}
+$$
+
+where:
+
+  - $p(y|\theta)$ is the likelihood,
+  - $p(\theta)$ is the prior distribution, and
+  - $p(y)$ is a normalising constant to ensure the posterior integrates to 1 over all $\theta$ values.
+
+Bayesian inference updates our beliefs about parameters based on observed data, yielding a posterior distribution that reflects both model assumptions and data influence. This probabilistic framework around a dynamical system’s parameters creates a flexible statistical model that adapts as new data becomes available.
+
+In this way, **stochastic processes** add an uncertainty dimension to the state of dynamical systems, while **Bayesian inference** applies probabilistic modelling to the parameters, merging dynamical and statistical perspectives. Together, these methods provide a rich toolkit for modelling complex systems that evolve over time and accounting for both randomness and uncertainty.
+
+## Parameters and model complexity
+
+### Parameters in dynamical systems
+
+In dynamical models, parameters define the structure and dynamics of the scenario. For example, a detailed epidemiological model may need parameters for transmission rates, contact patterns, or intervention schedules. These inputs allow us to examine "what-if" scenarios, enabling decision-makers to predict and manage changes in real-time.
+
+### Parameters in statistical models
+
+Statistical models, meanwhile, use parameters to define probability distributions over possible outcomes rather than scenario-specific details. Here, parameters describe uncertainties, often quantifying risks, making predictions, or summarising patterns in data.
+
+The `odin` package allows us to build and explore the behaviours of dynamical models over time, while `monty` enables us to quantify uncertainties and make probabilistic inferences without simulating detailed dynamics.
+
+## Bayesian or non-Bayesian modelling
+
+Many statistical modelling tools, such as `stan`, `mcstate`, and `BayesTools`, are inherently Bayesian. However, the `monty` package is **Bayesian-agnostic**: it does not require a Bayesian interpretation. This flexibility allows users to work with probabilistic models either within a Bayesian framework or outside of it, depending on the needs of the analysis.
+
+## Probability densities: normalised vs. unnormalised
+
+A key concept in Bayesian inference and Monte Carlo methods is the distinction between **normalised** and **unnormalised** probability densities.
+
+### Unnormalised density
```

Comment:
these are just paragraphs, not sections - it would read more smoothly without the headings I think

Reply:
Agree. Done.